compile_clang.sh

```diff
@@ -1,10 +1,10 @@
 #! /usr/bin/bash
 
-_cross_compile=$(realpath "$ANDROID_ROOT/prebuilts/gcc/linux-x86/aarch64/aarch64-linux-android-4.9/bin/aarch64-linux-android-")
-_cross_compile_32=$(realpath "$ANDROID_ROOT/prebuilts/gcc/linux-x86/arm/arm-linux-androideabi-4.9/bin/arm-linux-androideabi-")
+_cross_compile="$ANDROID_ROOT/prebuilts/gcc/linux-x86/aarch64/aarch64-linux-android-4.9/bin/aarch64-linux-android-"
+_cross_compile_32="$ANDROID_ROOT/prebuilts/gcc/linux-x86/arm/arm-linux-androideabi-4.9/bin/arm-linux-androideabi-"
 # Get the clang version from current AOSP documentation
 _clang_version=$(awk '/^\* \[\*\*Android Linux Kernel/{f=NR} /^  \* Currently clang-/ && f==NR-1 {print $NF; exit}' $ANDROID_ROOT/prebuilts/clang/host/linux-x86/README.md)
-_clang_path=$(realpath "$ANDROID_ROOT/prebuilts/clang/host/linux-x86/$_clang_version/bin")
+_clang_path="$ANDROID_ROOT/prebuilts/clang/host/linux-x86/$_clang_version/bin"
 echo "==> Using clang $_clang_path"
```

Comment:
I'd add a newline before the echo.

Reply:
I intended not to change irrelevant lines; this is going to change with #9 anyway.